### PR TITLE
Improve the mail header when using mail() function

### PR DESCRIPTION
**Pull request by truongwp**
This fixes this pro version issue https://github.com/Strategy11/formidable-pro/issues/2825

### CODE REVIEW

File: classes/models/FrmEmail.php [hunk 1]
```diff
@@ -435,7 +435,9 @@ private function send_single( $recipient ) {
 		$sent = wp_mail( $recipient, $subject, $this->message, $header, $this->attachments );
 
 		if ( ! $sent ) {
-			$header    = 'From: ' . $this->from . "\r\n";
+			if ( is_array( $header ) ) {
+				$header = implode( "\r\n", $header );
```
Comment (stephywells) [inline classes/models/FrmEmail.php:439]: Looks like we lost the "From: " part in here.... I think we'll need to keep that. Right?

Comment (truongwp) [reply classes/models/FrmEmail.php:439]: It's included here https://github.com/Strategy11/formidable-forms/blob/master/classes/models/FrmEmail.php#L466

Comment (truongwp) [reply classes/models/FrmEmail.php:439]: It's included here https://github.com/Strategy11/formidable-forms/blob/master/classes/models/FrmEmail.php#L466

Comment (stephywells) [reply classes/models/FrmEmail.php:439]: Oh, I see now! Thanks for clearing that up for me! Looks great!

**Review by stephywells — APPROVED** [review]

**Comment by stephywells** [comment]
@jairoprez This could use a quick test. The easiest way to test this is probably to change this line:
$sent = wp_mail( $recipient, $subject, $this->message, $header, $this->attachments );

to:
$sent = false;

Thanks for your help!

**Review by jairoprez — COMMENTED** [review]
@stephywells Thanks for the guide to test this PR, this fix seems to be working correctly.
- test with `wp_mail()`: https://recordit.co/ysRb6c0lpY
- test with `mail()`: https://recordit.co/YYiOnrwNrD

**Review by jairoprez — APPROVED** [review]

**Pull request merged**